Products/Zuul/facades/manufacturersfacade.py

```diff
@@ -261,6 +274,11 @@ def editManufacturer(self, params):
         if (manufacturer.id != params['name']):
             manufacturer.rename(params['name'])
 
+        # Set updated instances to the product
+        for pair in product_intance_pairs:
+            obj, prod = pair
+            obj.setProductClass(prod)
```

Comment:
Since `product_instance_pairs` contains pairs, this can be written as:
```
for obj, prod in product_instance_pairs:
    obj.setProductClass(prod)
```

Reply:
I Implemented the changes you suggested.